guides/dai_transaction_from_browser.md

```diff
@@ -0,0 +1,65 @@
+# Transacting DAI on OmiseGO Network through the browser
+
+Estimated time: 5 minutes
+
+Goal:
+
+- To understand plasma transactions with ERC20 token using DAI
+- To get first hand experience in making ERC20 interactions with plasma
+- To learn the process of making `deposit`, `transact` and `exit` with ERC20
+
+In this guide, we will introduce to you the Plasma Interface by making transactions directly from your browser. Just make sure you have the following installed:
+1. node-js >10.0.0
+2. npm > 6.0.0
+
+Have access to a code editor, browser with a web3 wallet like MetaMask
+
```

Comment:
**Overview**
This guide introduces you to the Plasma interface, and shows you how to make Plasma transactions directly from your browser, using DAI with ERC20 tokens.

You will learn first-hand how to use ERC20 interactions with Plasma to make a `deposit`, `transact`, and `exit`.

**Before you start**
 * Ensure you have the following installed: 
  * node-js >10.0.0
  * npm > 6.0.0
* You must have access to the following: 
  * A code editor
  * A browser with a web3 wallet, such as MetaMask